Source/JavaScriptCore/offlineasm/riscv64.rb

```diff
@@ -1332,9 +1423,9 @@ def emit(newList, node, precision, condition)
         operands = node.operands
 
         case condition
-        when :eq
+        when :eq, :equn
             emitCompare(newList, node, precision, "feq", operands[0], operands[1])
-        when :neq
+        when :neq, :nequn
```

Comment:
How are unordered FP values supposed to be handled here? Does `feq` do it for us in a way compatible with the intention of the LLInt instructions?